doc/releases/changelog-dev.md

```diff
@@ -82,6 +82,9 @@
   [(#4620)](https://github.com/PennyLaneAI/pennylane/pull/4620)
   [(#4632)](https://github.com/PennyLaneAI/pennylane/pull/4632)
 
+* `CosineWindow` has been added. Prepare the initial state following a cosine wave function.
+  [(#4683)](https://github.com/PennyLaneAI/pennylane/pull/4683)
+
```

Comment:
@KetpuntoG - not a blocker if things are tight with timing, but having a nice changelog entry with examples is very useful for the release.

Reply:
Okay, I should have time

Reply:
Thanks!